lib/SILGen/SILGenConvert.cpp

```diff
@@ -654,6 +654,40 @@ ManagedValue SILGenFunction::emitExistentialErasure(
     return emitManagedRValueWithCleanup(existential);
   }
   case ExistentialRepresentation::Opaque: {
+  
+    // If the concrete value is a pseudogeneric archetype, first erase it to
+    // its upper bound.
+    auto anyObjectProto = getASTContext()
+      .getProtocol(KnownProtocolKind::AnyObject);
+    auto anyObjectTy = anyObjectProto
+      ? anyObjectProto->getDeclaredType()->getCanonicalType()
```

Comment:
It's null in some of our SILGen tests that run without a standard library. The code crashed without handling null `anyObjectProto` and appears to work as is.